packages/ppx_yojson/ppx_yojson.1.0.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Nathan Rebours <nathan.p.rebours@gmail.com>"
+homepage: "https://github.com/NathanReb/ppx_yojson"
+bug-reports: "https://github.com/NathanReb/ppx_yojson/issues"
+license: "BSD-2"
+dev-repo: "git+https://github.com/NathanReb/ppx_yojson.git"
+doc: "https://nathanreb.github.io/ppx_yojson/doc"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test & (ocaml:version < "4.08" | ppxlib:version >= "0.8.1")}
+]
+depends: [
+  "dune" {build & >= "1.9.3"}
```

Comment:
```suggestion
  "dune" {>= "1.9.3"}
```
Dune shouldn't be tagged as a build dependency (see #14266)

Reply:
Ouh yeah, good catch. Thanks!